packs/duo/tests/fixtures/blank.yaml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
I've moved some of these fixtures, I'll attempt to work though any others as and when I see them.